functional-tests/README.md

```diff
@@ -129,6 +129,52 @@ For instance:
 
 As a result, services will be kept alive, so you can send RPCs and play around.
 
-To see the full list of supported envs as well as insights of each of them,
-navigate to `entry.py` and follow along.
+## Test Environment Configurations
+
+The functional tests support multiple environment configurations, each designed for specific testing scenarios:
+
+### **"basic"**
+- **Purpose**: Default environment for most tests
+- **Components**:
+  - Bitcoin regtest node
+  - Sequencer + Sequencer Signer
+  - Reth (Ethereum execution client)
+  - Prover Client
+- **Settings**: 110 pre-generated blocks, fast batch mode
+- **Use case**: Standard rollup functionality tests
+- **Example**: `./run_test.sh -e basic`
```

Comment:
Actually `-e` is used for "keep alive" mode in which the services related to given environment ast spun up and they just stay idle. This is for local setup and fast prototyping. This does not actually run the tests related to environment.

Reply:
Same with other examples. It should either be mentioned explicitly or the examples be removed. In fact, the behavior of `-e` is already explained just above this newly added section.